src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -12,9 +12,12 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Roslyn.Test.Utilities
 Imports Roslyn.Test.Utilities.TestBase
 Imports Xunit
+Imports Microsoft.CodeAnalysis.Collections
 
 Friend Module CompilationUtils
 
+    Private ReadOnly Property _pooledStringBuilderPool As ObjectPool(Of PooledStringBuilder) = PooledStringBuilder.CreatePool()
```

Comment:
>_pooledStringBuilderPool [](start = 30, length = 24)

The name sounds somewhat strange - pooled pool. If it is a pool of StringBuilders, then perhaps the name should be _stringBuilderPool?